python/pyarrow/tests/test_dataset.py

```diff
@@ -2810,11 +2819,11 @@ def test_parquet_dataset_lazy_filtering(tempdir, open_logging_fs):
 
     # filtering fragments should not open any file
     with assert_opens([]):
-        list(dataset.get_fragments(ds.field("f1") > 15))
+        list(dataset.get_fragments(ds.field("f2") > 15))
```

Comment:
The "f2" column is random normal around 0, so this filter will never yield anything

Reply:
I changed f2 to be int64 values 0, 10, 20, 30.  f1 is no longer usable because it got removed from the schema because it was a partition column.